src/test/java/org/folio/rest/api/HoldingsStorageTest.java

```diff
@@ -577,20 +577,158 @@ public void tenantIsRequiredForGettingAllHoldings()
+    assertThat(getSecondUpdatedItemResponse.getStatusCode(), is(HttpURLConnection.HTTP_OK));
+    assertThat(secondUpdatedItemFromGet.getString("id"), is(secondItemId));
+    assertThat(secondUpdatedItemFromGet.getString("effectiveCallNumber"), is("updatedCallNumber"));
+  }
+
+  @Test
+  public void clearingHoldingsCallNumberUpdatesItemEffectiveCallNumber()
+      throws InterruptedException, ExecutionException, TimeoutException, MalformedURLException {
+    UUID instanceId = UUID.randomUUID();
+
+    instancesClient.create(smallAngryPlanet(instanceId));
+
+    UUID holdingId = UUID.randomUUID();
+
+    JsonObject holding = holdingsClient.create(new HoldingRequestBuilder()
+      .withId(holdingId)
+      .forInstance(instanceId)
+      .withPermanentLocation(mainLibraryLocationId)
+      .withCallNumber("testCallNumber")
+      .withTags(new JsonObject().put("tagList", new JsonArray().add(TAG_VALUE)))).getJson();
+
+    JsonObject itemToCreate = new JsonObject();
+
+    itemToCreate.put("holdingsRecordId", holdingId.toString());
+    itemToCreate.put("status", new JsonObject().put("name", "Available"));
+    itemToCreate.put("permanentLoanTypeId", canCirculateLoanTypeID);
+    itemToCreate.put("temporaryLocationId", annexLibraryLocationId.toString());
+    itemToCreate.put("materialTypeId", bookMaterialTypeID.toString());
+
+    Response postFirstItemResponse = create(itemsStorageUrl(""), itemToCreate);
+
+    assertThat(postFirstItemResponse.getStatusCode(), is(HttpURLConnection.HTTP_CREATED));
+
+    JsonObject firstItemFromPost = postFirstItemResponse.getJson();
+
+    String firstItemId = firstItemFromPost.getString("id");
+
+    assertThat(firstItemId, is(notNullValue()));
+
+    URL getFirstItemUrl = itemsStorageUrl(String.format("/%s", firstItemId));
+
+    Response getFirstItemResponse = get(getFirstItemUrl);
+
+    JsonObject firstItemFromGet = getFirstItemResponse.getJson();
+
+    assertThat(firstItemFromGet.getString("id"), is(firstItemId));
+    assertThat(firstItemFromGet.getString("holdingsRecordId"), is(holdingId.toString()));
+    assertThat(firstItemFromGet.getString("effectiveCallNumber"), is("testCallNumber"));
+
+    URL holdingsUrl = holdingsStorageUrl(String.format("/%s", holdingId));
+
+    holding.remove("callNumber");
+
+    Response putResponse = update(holdingsUrl, holding);
+
+    assertThat(putResponse.getStatusCode(), is(HttpURLConnection.HTTP_NO_CONTENT));
+    assertThat(holding.containsKey("callNumber"), is(false));
+
+    Response getFirstUpdatedItemResponse = get(getFirstItemUrl);
+
+    JsonObject firstUpdatedItemFromGet = getFirstUpdatedItemResponse.getJson();
+
+    assertThat(getFirstUpdatedItemResponse.getStatusCode(), is(HttpURLConnection.HTTP_OK));
+    assertThat(firstUpdatedItemFromGet.getString("id"), is(firstItemId));
+    assertThat(firstUpdatedItemFromGet.getString("effectiveCallNumber"), is(""));
+  }
+
   public void cannotCreateHoldingWithoutPermanentLocation() throws Exception {
```

Comment:
May I ask you to add a `@Test` annotation for this test method? Seems it disappeared.